website/docs/docs/integrate/quickstart-semantic-layer.md

```diff
@@ -0,0 +1,292 @@
+
+1. Add the metric definitions found in the [Jaffle Shop](https://github.com/dbt-labs/jaffle_shop_metrics) example to your dbt project. For example, to add an Expenses metric, reference the following metrics you can define directly in your metrics folder: 
+      
+<VersionBlock firstVersion="1.3">
+
+```sql
+version: 2
+
+metrics:
+  - name: expenses
+    label: Expenses
+    model: ref('orders')
+    description: "The total expenses of our jaffle business"
+
+    calculation_method: sum
+    expression: amount / 4
+
+    timestamp: order_date
+    time_grains: [day, week, month, year]
+
+    dimensions:
+      - customer_status
+      - had_credit_card_payment
+      - had_coupon_payment
+      - had_bank_transfer_payment
+      - had_gift_card_payment
+
+    filters:
+      - field: status
+        operator: '='
+        value: "'completed'"
+```
+</VersionBlock> 
+
+<VersionBlock lastVersion="1.2">
+
+```sql
+version: 2
+
+metrics:
+  - name: expenses
+    label: Expenses
+    model: ref('orders')
+    description: "The total expenses of our jaffle business"
+
+    type: sum
+    sql: amount / 4
+
+    timestamp: order_date
+    time_grains: [day, week, month, year]
+
+    dimensions:
+      - customer_status
+      - had_credit_card_payment
+      - had_coupon_payment
+      - had_bank_transfer_payment
+      - had_gift_card_payment
+
+    filters:
+      - field: status
+        operator: '='
+        value: "'completed'"
+```
+</VersionBlock>       
+
+1. Click **Save** and **Compile** the code.
+2. Commit and merge the code changes that contain the metric definitions.
+3. If you'd like to further design and define your own metrics, review the following documentation:
+
+    - [dbt metrics](/docs/building-a-dbt-project/metrics) will povide you in-depth detail on attributes, properties, filters, and how to define and query metrics.
+   
+    - Visit the [Structure and design your metric definition](URL) page to understand best practices for designing and structuring metrics in your dbt project.
+
+## Develop and query metrics
+
+You can dynamically develop and query metrics directly in dbt and verify their accuracy *before* running a job in the deployment environment by using the `metrics.calculate` and `metrics.develop` macros.
+
+To understand when and how to use the macros above, review [dbt metrics](/docs/building-a-dbt-project/metrics) and make sure you install the [dbt_metrics package](https://github.com/dbt-labs/dbt_metrics) first before using the above macros.
+
+:::info 📌 
+
+**Note:** you will need access to dbt Cloud and the dbt Semantic Layer from your integrated tool of choice. 
+
+:::
+
+## Run your production job
+
+Once you’ve defined metrics in your dbt project, you can perform a job run in your deployment environment to materialize your metrics. The deployment environment is only supported for the dbt Semantic Layer at this moment. 
```

Comment:
Might be better to say "Only the deployment environment is supported for the dbt Semantic Layer at this moment."